simtools/applications/add_value_from_json_to_db.py

```diff
@@ -62,29 +69,47 @@ def main():
 
     db = db_handler.DatabaseHandler(mongo_db_config=db_config)
 
-    par_dict = gen.collect_data_from_file_or_dict(file_name=args_dict["file_name"], in_dict=None)
+    files_to_insert = []
+    if args_dict.get("file_name", None) is not None:
+        files_to_insert.append(args_dict["file_name"])
+    else:
+        files_to_insert.extend(Path(args_dict["input_path"]).glob("*json"))
```

Comment:
Probably intended, but just to make sure: This would add only files at the top level directory.

Reply:
Yes, the idea is to search only in "input_path" and not recursively.